tensorflow/python/BUILD

```diff
@@ -2570,7 +2571,9 @@ tf_py_wrap_cc(
         "//tensorflow/tools/graph_transforms:transform_graph_lib",
         "//tensorflow/tools/tfprof/internal:print_model_analysis",
         "//util/python:python_headers",
-    ] + tf_additional_lib_deps() + tf_additional_plugin_deps(),
+    ] + (tf_additional_lib_deps() +
+         tf_additional_plugin_deps() +
+         tf_additional_verbs_deps()),
```

Comment:
Because this code doesn't define a new op, this dependency is probably difficult to avoid. Likely we'll have to make an exception here.